services/QuillLMS/client/app/bundles/Teacher/components/scorebook/premium_banners/premium_banner_builder.jsx

```diff
@@ -68,31 +68,35 @@ export default class PremiumBannerBuilder extends React.Component {
     }
   };
 
-  stateSpecificBackGroundImage = () => {
-    const { daysLeft, } = this.props
-    if (daysLeft === 30){
-      return('none');
-    } else {
-      return('url(/images/star_pattern_5.png)');
-    }
-  };
-
   hasPremium = () => {
     const { has_premium, first_day_of_premium_or_trial, } = this.state
     let color = this.stateSpecificBackGroundColor();
-    let img = this.stateSpecificBackGroundImage();
     let divStyle = {
-      backgroundColor: color,
-      backgroundImage: img
+      backgroundColor: color
     };
     if ((has_premium === null) || (has_premium === 'school') || ((has_premium === 'paid') && (first_day_of_premium_or_trial === false))) {
       return (<span />);
     } else
     {
       return (
-        <div id='premium-banner' style={divStyle}>
-          <div className='container'>
-            {this.stateSpecificComponents()}
+        <div>
+          <div id='premium-banner' style={divStyle}>
+            <div className='container'>
+              {this.stateSpecificComponents()}
+            </div>
+          </div>
+          <div className='row school-premium-banner'>
```

Comment:
I know this is just based on the existing premium banner so it's nbd, but I would move away from using bootstrap styles like the rows and col widths - this design would be pretty easy to implement just using flexbox and require fewer wrapper elements.